geode-core/src/test/java/org/apache/geode/management/internal/cli/commands/DescribeConnectionCommandJUnitTest.java

```diff
@@ -49,13 +49,17 @@
   public GfshShellConnectionRule gfsh = new GfshShellConnectionRule();
 
   @Test
-  public void executeWhileConnected() throws Exception {
-    // Depending on configuration, the host may resolve its local IP
-    // to the alias present in /etc/hosts or use its internal host name.
-    // Check for these first.
-    gfsh.connectAndVerify(locator);
-    gfsh.executeAndAssertThat("describe connection")
-        .tableHasColumnWithValuesContaining("Connection Endpoints", gfsh.getGfsh().getOperationInvoker().toString());
+  public void describeJmxConnection() throws Exception {
+    gfsh.connectAndVerify(locator.getJmxPort(), GfshShellConnectionRule.PortType.jmxManager);
+    gfsh.executeAndAssertThat("describe connection").tableHasColumnWithValuesContaining(
+        "Connection Endpoints", gfsh.getGfsh().getOperationInvoker().toString());
+  }
+
+  @Test
+  public void describeHttpConnection() throws Exception {
```

Comment:
if this test is not in geode-web or geode-assembly, I think this would fail. I don't think it would necessary to test http. the assertions are all the same.